assets/base.css

```diff
@@ -2539,7 +2539,9 @@ details-disclosure > details {
 }
 
 .global-media-settings img,
-.global-media-settings iframe {
+.global-media-settings iframe,
+.global-media-settings model-viewer,
+.global-media-settings video {
   border-radius: calc(var(--media-radius) - var(--media-border-width));
```

Comment:
Should it also apply on thumbnails for `img` ? 🤔 
[screenshot](https://screenshot.click/22-01-4bkus-7g88t.png)

Maybe not something to look into in this PR but the one its using as origin.